### DISCUSSION TITLE
Mendapatkan respon webhook dari pengguna API lain

### DISCUSSION THREAD

**Discussion by shahib-mirbath** [kirimwa-id/forum#124] [Tanya Jawab]
Lapor

Saya mendapatkan (lagi) webhook respon dari pengguna api.kirimwa pada 06-08-2022 jam 10.00 dan 11:34 WIB. Id pesan yg dikirim: kwid-*** dan kwid-***

Terimakasih

**Comment by rioastamal** [comment]
@shahib-mirbath Apa tipe dari webhook yang diterima `send_message_response` atau yang lain?

**Reply by shahib-mirbath** [reply]
Sementara ini yg saya dapat adalah send_message_response

**Comment by masbagusstwn** [comment]
> @shahib-mirbath Apa tipe dari webhook yang diterima `send_message_response` atau yang lain?

saya juga dapat tiba2 mas rio, sudah 2x dalam sebulan ini hehe. pertama saat send message terus webhook yg masuk malah beda. yg kedua gak ada  aktifitas apa apa tetiba masuk webhook send_message_response dari requester lain.. dan isinya kayaknya dua duanya tipe spam deh mas, bahaya tuu

**Reply by rioastamal** [reply]
Mohon kirim Message ID dari yang ada dari webhook `send_message_response` tersebut, biar bisa dianalisa apakah memang SPAM.
